pipelines/tagger_parser_ud/project.yml

```diff
@@ -26,16 +26,14 @@ assets:
 
 workflows:
   all:
-    - convert
+    - preprocess
```

Comment:
Good to have this sync'ed better with the spaCy docs, indeed.

Could you update the readme file as well? If you run the `update_docs.py` script from within the `.github` folder, it will automatically update all readme files. It might catch a few other outdated ones as well, that's fine ;-)

Reply:
Ah, got it - when I heard the script was in `.github` I thought it ran automatically :sweat_smile:

Reply:
We probably could ;-) Then again I think there could be cases where you need to double check / adjust manually as well... Anyway, thanks! Merging :-)